src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -3282,10 +3282,10 @@ static void Main()
     }
 }
 ";
-            CreateCompilation(source);
+            CreateCompilation(source).GetDiagnostics();
```

Comment:
Rather than `GetDiagnostics`, can we do `VerifyDiagnostics`?

Reply:
The comment it generates is very long (since it contains the full line with 3000 tuple members) but I guess I can exclude that line.